models/class.metrilo-deleted-product-data.php

```diff
@@ -0,0 +1,99 @@
+<?php
+class Metrilo_Deleted_Product_Data
+{
+    private $db_connection;
+    
+    public function __construct()
+    {
+        global $wpdb;
+        
+        $this->db_connection = $wpdb;
+    }
+    
+    public function get_deleted_products()
+    {
+        $deleted_products        = [];
+        $deleted_product_options = [];
+        $parent_id = '';
+        $child_id  = '';
+        $qty       = '';
+        $subtotal  = '';
```

Comment:
Why do you define those here?
For example - what will happen if you set `$parent_id` in the first iteration of the `foreach` (line 40) and then on a second iteration - `$parent_id` will be what value?